src/telecom/voip/index.js

```diff
@@ -2,6 +2,8 @@ import angular from 'angular';
 
 import 'ovh-api-services';
 
+import { TUC_TELEPHONY_ALIAS } from './telecom-voip.constant';
```

Comment:
```suggestion
import { TELEPHONY_ALIAS } from './telecom-voip.constant';
```

We don't need to prefix constants with `TUC_`, we only should prefix them when we call `angular.constant` function.